src/components/autosuggest/_macro.spec.js

```diff
@@ -81,6 +81,10 @@ describe('FOR: Macro: Autosuggest', () => {
                 expect($('.ons-autosuggest').attr('data-result-threshold')).toBe('0.5');
             });
 
+            test('THEN: it has language set to "en-gb"', () => {
+                expect($('.ons-autosuggest').attr('data-lang')).toBe('en-gb');
+            });
```

Comment:
Only required params should be inside this "when" unless you are testing defaults. I don't think this param is required and theres probably a few others too such as the result threshold one. I think we need to review these, keep the ones that test defaults move the others. Then for the ones that have defaults make sure the defaults are tested here and have another test to test what you pass in is overriding that default

Reply:
You've moved the two params I mentioned, but haven't reviewed the others. Theres still some params in there that aren't required